pkg/storage/unified/resource/keys.go

```diff
@@ -28,3 +33,39 @@ func matchesQueryKey(query *ResourceKey, key *ResourceKey) bool {
 	}
 	return true
 }
+
+const clusterNamespace = "**cluster**"
```

Comment:
what does this mean? Isn't on-prem the only case that namespace is empty? And then in that case we should use orgID as the namespace?

Reply:
this case should not come up often (ever?) for resources that do not belong to a tenant, the namespace will be empty.  In k8s these are called "cluster scope"

But we need to put something in the value so the prefix is different!